### PR TITLE
chore: Passing NPM_Token into Build/Release yml file

### PR DESCRIPTION
**Pull request by akinsola-guardian**
## What does this change?
- Passing NPM token into workflow_call
- Updating permission
- Updating CI Monitoring permissions.
## Why?
- Fix beta deploy

### CODE REVIEW

File: .github/workflows/monitoring-CI.yml [hunk 1]
```diff
@@ -71,6 +72,9 @@ jobs:
 
       - name: Upload to Riff Raff
         uses: guardian/actions-riff-raff@v3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (sookburt) [inline .github/workflows/monitoring-CI.yml:77]: Does the monitoring release need the NPM_TOKEN if it 's not deployed to npm?

Comment (akinsola-guardian) [reply .github/workflows/monitoring-CI.yml]: Thanks, I just removed this.

File: .github/workflows/cmp-beta-release-on-label.yml [hunk 2]
```diff
@@ -21,8 +21,11 @@ jobs:
...
     needs: [run-tests]
     uses: ./.github/workflows/cmp-build-release.yml
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
     permissions:
       checks: write
       pull-requests: write
       contents: write
       id-token: write
+      issues: write
```
Comment (sookburt) [inline .github/workflows/cmp-beta-release-on-label.yml:31]: I notice this has a `permissions: write-all` at line 10 - is that required now we have more fine grained permissions in the jobs themselves?

Comment (akinsola-guardian) [reply .github/workflows/cmp-beta-release-on-label.yml:36]: Thanks. I've updated this and tested to check if it affects the beta deploy.

**Review by sookburt — APPROVED** [review]
Looking good!

**Pull request merged**